src/main/java/io/lyuda/jcards/Dice.java

```diff
@@ -0,0 +1,52 @@
+package io.lyuda.jcards;
+
+import java.security.SecureRandom;
+
+/**
+ * The {@code Dice} class represents a dice with an arbitrary amount of sides, defined in the constructor.
+ * This class contains methods to access the number of sides, as well as rolling the dice to generate a random number.
+ *
+ * @author KonnosBaz
+*/
+public class Dice {
```

Comment:
Naming:

"Dice" is the *plural" of "Die" - a single Die and many Dice - there are no "dices".

For a single Die, the class name should be "Die", not "Dice"

Reply:
I can still change it if you disagree